controller/space_iterations_test.go

```diff
@@ -103,6 +104,49 @@ func (rest *TestSpaceIterationREST) TestSuccessCreateIteration() {
 	require.NotNil(rest.T(), c.Data.Relationships.Space)
 	assert.Equal(rest.T(), p.ID.String(), *c.Data.Relationships.Space.Data.ID)
 	assert.Equal(rest.T(), iteration.StateNew.String(), *c.Data.Attributes.State)
+	assert.False(rest.T(), *c.Data.Attributes.UserActive)
+	assert.Equal(rest.T(), "/"+rootItr.ID.String(), *c.Data.Attributes.ParentPath)
+	require.NotNil(rest.T(), c.Data.Relationships.Workitems.Meta)
+	assert.Equal(rest.T(), 0, c.Data.Relationships.Workitems.Meta[KeyTotalWorkItems])
+	assert.Equal(rest.T(), 0, c.Data.Relationships.Workitems.Meta[KeyClosedWorkItems])
+}
+
+func (rest *TestSpaceIterationREST) TestSuccessCreateIterationWithForceActive() {
+	// given
+	var p *space.Space
+	var rootItr *iteration.Iteration
+	ci := createSpaceIteration("Sprint #21", nil)
+	ci.Data.Attributes.UserActive = ptr.Bool(true)
+	err := application.Transactional(rest.db, func(app application.Application) error {
+		repo := app.Spaces()
+		newSpace := space.Space{
```

Comment:
@baijum Please use the test fixture to create a space. You're not doing anything special here, so the test fixture should be used.